doc/source/cluster/kuberay/quickstart.ipynb

```diff
@@ -0,0 +1,616 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f34f1b75",
+   "metadata": {},
+   "source": [
+    "(kuberay-quickstart)=\n",
+    "\n",
+    "# Getting Started\n",
+    "\n",
+    "In this guide, we discuss how to manage and interact with Ray clusters on Kubernetes.\n",
+    "\n",
+    "You can download this guide as an executable Jupyter notebook by clicking the download button on the top right of the page.\n",
+    "\n",
+    "\n",
+    "## Preparation\n",
+    "\n",
+    "### Install the latest Ray release"
```

Comment:
This is only needed if the user is using the Ray client, right? If yes let's shift it to that section, otherwise it might be confusing (since the Ray version is determined from the K8s manifests and not the locally installed Ray version).

Reply:
You need for job submission too. I think we might as well have people install Ray.

Reply:
Then maybe add a sentence at the beginning that this is for using the Ray Client and Job submission or similar?

Reply:
Will do.